### DISCUSSION TITLE
Support policy for Kong Gateway Community edition

### DISCUSSION THREAD

**Discussion by shazib-summar** [Kong/kong#13418] [Help]
Hello. I'm confused by the table at https://docs.konghq.com/gateway/latest/support-policy/#supported-versions

Does this table apply for the Kong API Gateway Community edition too? Confused because at the top there is a info box that says "This policy only applies to Kong Gateway and Kong Gateway Enterprise." 

<img width="788" alt="image" src="https://github.com/user-attachments/assets/5f9bb1b2-d945-409f-89e9-dc2fb260183b">

but at the section "Supported versions" it says "Kong supports the following versions of Kong Gateway Enterprise"?

<img width="563" alt="image" src="https://github.com/user-attachments/assets/0837c787-5f1c-4c3d-8249-9e6644173975">

Where can I find the list of supported versions for Kong Gateway Community edition? Would appreciate any help on this.

**Comment by guanlan** [comment]
We only offer limited backport support for the OSS version on high-severity security issues at our discretion. 

OSS users are strongly encouraged to upgrade to the latest version to receive fixes and the newest features.

**Reply by shazib-summar** [reply]
@guanlan thanks for reaching out. Sorry but i'm still somewhat confused. Kong 2.8 is being maintained despite not being the latest release: https://github.com/Kong/kong/releases

Looking at the [Supported versions](https://docs.konghq.com/gateway/latest/support-policy/#supported-versions) section, it appears that Kong 2.8 is an LTS release. Do Kong OSS get LTS as well?

**Reply by shazib-summar** [reply]
Frankly the entire page at https://docs.konghq.com/gateway/latest/support-policy/ is a bit confusing. I cannot tell which parts in that doc apply to Kong OSS and which apply to Kong Enterprise.

Take the page title for example : "Kong Gateway Enterprise Version Support" which gives the idea that this page applies to Kong Enterprise. But the "Note" banner underneath is contradictory suggesting that this applies to Kong OSS too: "This policy only applies to Kong Gateway and Kong Gateway Enterprise"

Then the Supported versions section says "Kong supports the following versions of Kong Gateway Enterprise:". But below says "Kong Gateway 2.8 LTS supports ....." which suggests Kong Gateway OSS since 
- its not explicitly called out "Kong Gateway Enterprise 2.8 LTS"
- the [GitHub releases](https://github.com/Kong/kong/releases) show that Kong Gateway OSS 2.8 is indeed supported
